### DISCUSSION TITLE
Conflict with Simone Franco's WSATools

### DISCUSSION THREAD

**Discussion by weareblahs** [weareblahs/WSAUtilities#10] [General]
Actually, that time I'm unaware of this APK installer until I did a Google search around this mid-November. Turns out that the APK installer called WSATools was released on 

This project is originally called `wsa-auto-install` then `WSAtools`. I think the capitalization can make clear for these stuffs, Also, I linked the APK installer on the README.md page to inform users about this.

@Simizfo: just curious, any opinions, or any name change request for my repo? Won't use `wsa-toolbox` or `WSAtoolbox` because of major confusion with another project with the same name.

**Comment by Simizfo** [comment]
Oh wow, hello there! I had no idea of this repo existing.

I've seen that your project ~~got created on November 2, based on your first commit,~~ (edit down in the post) while mine has had WSATools as name since October 21, when I [posted the first tweet on Twitter](https://twitter.com/Simizfo/status/1451216318338850816). So, yes, sorry if this sounds pretentious, but I'd like you to change your project name. I want to keep mine, since I was the first one to take it. 

I can't, of course, force you to do it, but I guess you understand that this would be bad for both of us. Both of our users, searching for "wsatools", will find either mine or your project, and have a 50% chance of finding the wrong one (both if they're searching for mine, or for yours).

I also understand that WSATools name fits your project better than mine, at the moment, but I plan on adding more features, so it will fit me in the future too. 

I've also got a domain already, [wsatools.app](wsatools.app), so that would be a waste of money too... 

**Edit: My bad, your first commit is on October 21 too. That makes us even! I stand on my position, though. I've got a bunch of websites reporting on my app with this name, so it makes sense that I'd keep it. I've also gone through a bunch of mails with Microsoft to keep using it, since they didn't want that...**

**Reply by weareblahs** [reply]
According to my research: one of the websites introducing WSATools links the Microsoft Store link to your WSATools while the GitHub link to my WSAtools, although the screenshot is from your WSATools.

Also, about the name change stuffs: I'm OK with that - also planned to do that soon. Just need your opinion, is the name `TheWSAToolbox` suitable for this project?

**Reply by Simizfo** [reply]
Yeah. As there's already some confusion, we need to avoid this, it won't benefit any one of us.

I'm thankful for you being cooperative on this. I promise I'll spend some time thinking about alternative names for your project. There's already a project named WSAToolbox, so you might get into issues with them if you choose `TheWSAToolbox`.

I'll reply here when I get some ideas.

**Reply by weareblahs** [reply]
Sure.

**Reply by Simizfo** [reply]
Here's a few ideas I've come up with:

`WSAKit`
`WSAUtilities`
`WSAManager`

Or you might go with something more particular: Someone on Twitter in the past suggested me `Mocha` - if you remember, the internal codename for WSA was `Latte`, so it makes sense for an tools/helper app to be called as a darker version of Latte

**Reply by weareblahs** [reply]
I think WSAUtilities could be suitable for my project. That name seems suitable for my own project, as it is a set of utilities for WSA. Thanks!

**Reply by Simizfo** [reply]
Thanks to you for your cooperation! 
Good luck with your project :)
